src/set_theory/game/ordinal.lean

```diff
@@ -13,14 +13,16 @@ import set_theory.ordinal.basic
 We define the canonical map `ordinal → pgame`, where every ordinal is mapped to the game whose left
 set consists of all previous ordinals.
 
+The map to surreals is defined in `ordinal.to_surreal`.
+
 # Main declarations
 
 - `ordinal.to_pgame`: The canonical map between ordinals and pre-games.
 - `ordinal.to_pgame_embedding`: The order embedding version of the previous map.
 
 # Todo
 
-- Extend this map to `game` and `surreal`.
+- Extend this map to `game`.
```

Comment:
Doesn't this overlap with the `-- TODO define the inclusion of groups `surreal → game` visible in the other file? I don't think we need `A.to_C` if we have `A.to_B` and `B.to_C`.

Reply:
True, I'll remove this Todo entirely.